pyt/cfg/expr_visitor.py

```diff
@@ -114,11 +114,10 @@ def visit_Yield(self, node):
         label = LabelVisitor()
         label.visit(node)
 
-        try:
-            rhs_visitor = RHSVisitor()
-            rhs_visitor.visit(node.value)
-        except AttributeError:
-            rhs_visitor.result = 'EmptyYield'
+        if node.value is None:
```

Comment:
You write such high quality code, but the amazing commit messages is what really separates you from the rest.